### PR TITLE
feat: apiref pages

### PR DESCRIPTION
**Pull request by victormendoza96**
image

### CODE REVIEW

File: src/app/api-reference/transaccion-completa-diferido/ApiRefTxDeferredCapture.tsx [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+  };
+
+  return (
+    <Card className="card-pay">
+      <span className="large-title">transaction.capture()</span>
+      <form action={handleSubmit}>
+        <div className="inp-container-card">
+          <InputText name="token" label="Token" />
+          <InputText name="buyOrder" label="Orden de compra" />
+          <InputText name="authorizationCode" label="Codigo de autorización" />
```
Comment (mastudillot) [inline src/app/api-reference/transaccion-completa-diferido/ApiRefTxDeferredCapture.tsx]:
```suggestion
          <InputText name="authorizationCode" label="Código de autorización" />
```

File: src/app/api-reference/oneclick-mall-deferred/ApiRefOneclickDeferredCapture.tsx [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+        <div className="inp-container-card">
+          <InputText
+            name="childBuyOrder"
+            label="Orden de compra(tienda hija)"
+          />
+          <InputText
+            name="commerceCode"
+            label="Codigo de comercio(tienda hija)"
+          />
+          <InputText name="authorizationCode" label="Codigo de autorización" />
```
Comment (mastudillot) [inline src/app/api-reference/oneclick-mall-deferred/ApiRefOneclickDeferredCapture.tsx]:
```suggestion
          <InputText name="authorizationCode" label="Código de autorización" />
```

File: src/app/api-reference/transaccion-completa-mall-diferido/ApiRefTxCompleteMallRefund.tsx [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+
+  return (
+    <Card className="card-pay">
+      <span className="large-title">transaction.refund()</span>
+      <form action={handleSubmit}>
+        <div className="inp-container-card">
+          <InputText name="token" label="Token" />
+          <InputText
+            name="commerceCode"
+            label="Codigo de comercio(tienda hija)"
```
Comment (mastudillot) [inline src/app/api-reference/transaccion-completa-mall-diferido/ApiRefTxCompleteMallRefund.tsx]:
```suggestion
            label="Código de comercio tienda"
```

File: src/app/api-reference/transaccion-completa-mall-diferido/ApiRefTxCompleteMallRefund.tsx [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+          />
+          <InputText name="buyOrder" label="Order de compra(tienda hija)" />
```
Comment (mastudillot) [inline src/app/api-reference/transaccion-completa-mall-diferido/ApiRefTxCompleteMallRefund.tsx]:
```suggestion
          <InputText name="buyOrder" label="Order de compra tienda" />
```

File: src/app/api-reference/transaccion-completa-mall-diferido/ApiRefTxMallDeferredCapture.tsx [hunk 5]
```diff
@@ -0,0 +1,69 @@
...
+
+  return (
+    <Card className="card-pay">
+      <span className="large-title">transaction.capture()</span>
+      <form action={handleSubmit}>
+        <div className="inp-container-card">
+          <InputText name="token" label="Token" />
+          <InputText
+            name="commerceCode"
+            label="Codigo de comercio(Tienda hija)"
```
Comment (mastudillot) [inline src/app/api-reference/transaccion-completa-mall-diferido/ApiRefTxMallDeferredCapture.tsx]:
```suggestion
            label="Código de comercio Tienda"
```

File: src/app/api-reference/transaccion-completa-mall-diferido/ApiRefTxMallDeferredCapture.tsx [hunk 6]
```diff
@@ -0,0 +1,69 @@
...
+          />
+          <InputText name="buyOrder" label="Orden de compra(Tienda hija)" />
```
Comment (mastudillot) [inline src/app/api-reference/transaccion-completa-mall-diferido/ApiRefTxMallDeferredCapture.tsx]:
```suggestion
          <InputText name="buyOrder" label="Orden de compra Tienda" />
```

File: src/app/api-reference/transaccion-completa-mall-diferido/ApiRefTxMallDeferredCapture.tsx [hunk 7]
```diff
@@ -0,0 +1,69 @@
...
+          <InputText name="authorizationCode" label="Codigo de autorización" />
```
Comment (mastudillot) [inline src/app/api-reference/transaccion-completa-mall-diferido/ApiRefTxMallDeferredCapture.tsx]:
```suggestion
          <InputText name="authorizationCode" label="Código de autorización" />
```

File: src/app/api-reference/transaccion-completa-mall-diferido/page.tsx [hunk 8]
```diff
@@ -0,0 +1,83 @@
...
+          Las transacciones de Webpay se pueden anular o reversar dadas algunas
+          condiciones. Para cualquiera de éstas operaciones se utiliza el mismo
+          servicio web que discernirá si se realizará una reversa o una
+          anulación. para mas informacion sobre anulaciones a reversa visite la{" "}
+          <a
+            href="https://www.transbankdevelopers.cl/producto/webpay#anulaciones-y-reversas"
+            target="_blank"
+            className="tbk-link"
+          >
+            documentacion
```
Comment (mastudillot) [inline src/app/api-reference/transaccion-completa-mall-diferido/page.tsx:73]:
```suggestion
            documentación
```

File: src/app/api-reference/transaccion-completa-mall/ApiRefTxCompleteMallRefund.tsx [hunk 9]
```diff
@@ -0,0 +1,68 @@
...
+
+  return (
+    <Card className="card-pay">
+      <span className="large-title">transaction.refund()</span>
+      <form action={handleSubmit}>
+        <div className="inp-container-card">
+          <InputText name="token" label="Token" />
+          <InputText
+            name="commerceCode"
+            label="Codigo de comercio(tienda hija)"
```
Comment (mastudillot) [inline src/app/api-reference/transaccion-completa-mall/ApiRefTxCompleteMallRefund.tsx]:
```suggestion
            label="Código de comercio tienda"
```

File: src/app/api-reference/transaccion-completa-mall/ApiRefTxCompleteMallRefund.tsx [hunk 10]
```diff
@@ -0,0 +1,68 @@
...
+          />
+          <InputText name="buyOrder" label="Order de compra(tienda hija)" />
```
Comment (mastudillot) [inline src/app/api-reference/transaccion-completa-mall/ApiRefTxCompleteMallRefund.tsx]:
```suggestion
          <InputText name="buyOrder" label="Order de compra tienda" />
```

File: src/app/api-reference/webpay-mall-deferred/ApiRefWebpayMallDeferredCapture.tsx [hunk 11]
```diff
@@ -0,0 +1,69 @@
...
+
+  return (
+    <Card className="card-pay">
+      <span className="large-title">transaction.capture()</span>
+      <form action={handleSubmit}>
+        <div className="inp-container-card">
+          <InputText name="token" label="Token" />
+          <InputText
+            name="commerceCode"
+            label="Codigo de comercio(Tienda hija)"
```
Comment (mastudillot) [inline src/app/api-reference/webpay-mall-deferred/ApiRefWebpayMallDeferredCapture.tsx]:
```suggestion
            label="Código de comercio tienda"
```

File: src/app/api-reference/webpay-mall-deferred/ApiRefWebpayMallDeferredCapture.tsx [hunk 12]
```diff
@@ -0,0 +1,69 @@
...
+          />
+          <InputText name="buyOrder" label="Orden de compra(Tienda hija)" />
```
Comment (mastudillot) [inline src/app/api-reference/webpay-mall-deferred/ApiRefWebpayMallDeferredCapture.tsx]:
```suggestion
          <InputText name="buyOrder" label="Orden de compra tienda" />
```

File: src/app/api-reference/webpay-mall-deferred/ApiRefWebpayMallDeferredRefund.tsx [hunk 13]
```diff
@@ -0,0 +1,68 @@
...
+
+  return (
+    <Card className="card-pay">
+      <span className="large-title">Formulario de redirección</span>
+      <form action={handleSubmit}>
+        <div className="inp-container-card">
+          <InputText name="token" label="Token" />
+          <InputText
+            name="commerceCode"
+            label="Codigo de comercio(tienda hija)"
```
Comment (mastudillot) [inline src/app/api-reference/webpay-mall-deferred/ApiRefWebpayMallDeferredRefund.tsx]:
```suggestion
            label="Código de comercio tienda"
```

File: src/app/webpay-mall/content/steps/refund.tsx [hunk 14]
```diff
@@ -15,10 +15,10 @@ export const getRefundTRXSteps = (
           <Text>
             Para llevar a cabo el reembolso, necesitas proporcionar el token de
             la transacción, el monto que quieres reversar, el código de comercio
-            de la tienda hijo y el orden de compra del detalle de la
-            transacción. Si anulas el monto total, podría ser una Reversa o
-            Anulación, dependiendo de ciertas condiciones, o una Anulación
-            Parcial si el monto es menor al total.
+            de la tienda y el orden de compra del detalle de la transacción. Si
```
Comment (mastudillot) [inline src/app/webpay-mall/content/steps/refund.tsx:18]:
```suggestion
            de la tienda y la orden de compra del detalle de la transacción. Si
```

**Review by mastudillot — COMMENTED** [review]
Check my comments

**Review by DiegoHBustamante — APPROVED** [review]

**Pull request merged**